### PR TITLE
Refactor resource-request-controller tests

### PR DESCRIPTION
**Pull request by palexster**
# Description

This PR:

* [x] refactors the resource-request-controller tests.
* [x] Moves the resource-request controller to the right controller-manager package.

# Tests

- [x] Unit Tests (existent)
- [x] E2E (existent)

### CODE REVIEW

File: deployments/liqo/files/liqo-controller-manager-ClusterRole.yaml [hunk 1]
```diff
@@ -262,6 +262,25 @@ rules:
   - get
   - patch
   - update
+- apiGroups:
```
Comment (giorio94) [inline deployments/liqo/files/liqo-controller-manager-ClusterRole.yaml:265]: Wondering why this permissions weren't present before...

**Review by giorio94 — APPROVED** [review]

**Review by aleoli — APPROVED** [review]

**Review by giuse2596 — APPROVED** [review]

File: deployments/liqo/files/liqo-controller-manager-ClusterRole.yaml (hunk 1 above)
Comment (palexster) [reply deployments/liqo/files/liqo-controller-manager-ClusterRole.yaml:265]: I think those permissions should be defined elsewhere. Investigating...

Comment (palexster) [reply deployments/liqo/files/liqo-controller-manager-ClusterRole.yaml:265]: It's just an issue of misspelled case: resourceOffers, resourceRequest.... Fixing.

**Pull request merged**